src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/LocalizedDigitsKeyListener.cs

```diff
@@ -1,32 +1,31 @@
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using Android.Text;
 using Android.Text.Method;
 using Java.Lang;
 using Java.Text;
 
-namespace Xamarin.Forms.Platform.Android
+namespace Xamarin.Platform
 {
-	internal class LocalizedDigitsKeyListener : NumberKeyListener
+	public class LocalizedDigitsKeyListener : NumberKeyListener
 	{
 		readonly char _decimalSeparator;
 
 		// I'm not aware of a situation/locale where this would need to be something different, 
 		// but we'll make it easy to localize the sign in the future just in case
 		const char SignCharacter = '-';
 
-		static Dictionary<char, LocalizedDigitsKeyListener> s_unsignedCache;
-		static Dictionary<char, LocalizedDigitsKeyListener> s_signedCache;
+		static Dictionary<char, LocalizedDigitsKeyListener>? UnsignedCache;
+		static Dictionary<char, LocalizedDigitsKeyListener>? SignedCache;
 
-		static char GetDecimalSeparator()
+		static char? GetDecimalSeparator()
```

Comment:
We should not allow this to be `null`. I honestly don't know why DecimalFormatSymbols is even nullable, but since it is this should be something more like 

```
		static char GetDecimalSeparator()
		{
			if (!(NumberFormat.Instance is DecimalFormat format))
			{
				return '.';
			}

			DecimalFormatSymbols? sym = format.DecimalFormatSymbols;
			return sym?.DecimalSeparator ?? '.';
		}
```